src/pages/ProductListing/ProductListing.css

```diff
@@ -35,21 +35,21 @@
   color: var(--dark-gray);
 }
 
-.prod-action-btn:disabled{
-  background-color:var(--dark-gray);
+.prod-action-btn:disabled {
+  background-color: var(--dark-gray);
```

Comment:
if possible try to name color by element where you are using it. avoid using direct color names as variables. or even if you want to use a color name then specify the element before it. 
Attaching screenshot of Github's color variables (This is from the same page of code review)
![image](https://user-images.githubusercontent.com/77927918/162576262-2d1d4adf-66d9-45f5-a14a-ffaa00458bc3.png)